### PR TITLE
release: v2.13.2

### PR DESCRIPTION
**Pull request by jonathanadler**
# ✨ Pull Request

### 💁 Description

...

### 💭 Motivation

...

### ☝️ TODOs 

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_541/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## [2.13.2](https://github.com/wix/wix-ui-tpa/compare/v2.13.1...v2.13.2) (2020-12-20)
+
+
```
Comment (sivanharel) [inline CHANGELOG.md:3]:
```suggestion
### Bug Fixes

"fix(TextArea): Use the default backgroundColor param״ ([c33b4d2](https://github.com/wix/wix-ui-tpa/commit/c33b4d2d0e8e4c934f3152eea9b9e1a921bfafff))


```

Comment (sivanharel) [reply CHANGELOG.md:3]: can you add it, please?

**Review by sivanharel — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jonathanadler) [reply CHANGELOG.md:5]: added

**Pull request merged**